### PR TITLE
rework `check` and rename to `proc`

### PR DESCRIPTION
**Pull request by nerdypepper**
remove file action, normalize dependency paths

### CODE REVIEW

**Comment by nerdypepper** [comment]
on hold until we merge history management.

File: server/bleep/src/webserver/answer.rs
Comment (calyptobai) [inline server/bleep/src/webserver/answer.rs:454]: Leftover `dbg!` statement.

**Review by calyptobai — APPROVED** [review]
LGTM :+1: Left a nit for an old `dbg!` call

**Pull request merged**
